Woof/Woof/ViewModel/Owner/BookingSlotViewModel.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+/// Responsible for preparing and processing data for the `BookingSlotView`.
+final class BookingSlotViewModel: ObservableObject {
+    /// The notes about any details of the walking like meeting place, pet behavior, etc.
+    @Published var notes: String = ""
+
+    /// The full name of the sitter.
+    let fullName: String
```

Comment:
I think all these properties can be computed and thus we can simplify init
```suggestion
    var fullName: String {
        DataTransformer.fullName(name: walking.sitter.name, surname: walking.sitter.surname)
    }
```

Reply:
fixed with https://github.com/IronFoundation/iOS-Woof/pull/112/commits/2e828042bd82cd3fd986fcac8f6eef903d62124e